docs/operations_guide/Lite.md

```diff
@@ -71,6 +71,10 @@ Each k8s pod representing runtime container is visble in metrics tab as separate
 Runtime container logging level can be specified by setting env variable
 - `NUSSKNACKER_LOG_LEVEL` - logging level of console STDOUT appender
 
+If you need more fine-grained control over logging you can provide your own `logback.xml` config file by overriding config map linked to your runtime container under `logback.xml` key.
```

Comment:
I think we should make it clear, that:
- overriding scenarios config map is transient (e.g. after redeployment the changes will return to defaults)
- shared configmap is persistent - redeployments won't change it